src/SamplesApp/UITests.Shared/Windows_UI_Xaml/Performance/Performance_ManyImages.xaml.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Xaml;
+using Uno.UI.Samples.Controls;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Media.Imaging;
+using SamplesApp.UITests;
+using CollectionExtensions = Uno.Extensions.CollectionExtensions;
+
+namespace UITests.Windows_UI_Xaml.Performance
+{
+	[Sample("Performance")]
```

Comment:
Let's make this one a manual test with explanations on what to observe. Also, as we're doing mip-mapping, it would be useful to have a test that contains a scrollviewer with some zoom feature (or manual zoom test with render transforms)